app/views/quotes/_quote.html.erb

```diff
@@ -1,23 +1,35 @@
 <li class="py-4">
   <div class="flex space-x-3 text-sm">
-    <%= link_to quote.user, class: "self-center" do %>
-      <% gravatar_for quote.user, class: "h-8 w-8 rounded-full" %>
+    <% if quote.anonymous %>
+      <div class="self-center">
+        <%= image_tag 'anonymous.svg', class: "w-7 h-7 rounded-full" %>
+      </div>
+    <% else %>
+      <%= link_to quote.user, class: "self-center" do %>
+        <% gravatar_for quote.user, class: "h-8 w-8 rounded-full" %>
+      <% end %>
     <% end %>
     <div class="flex-col flex-1">
       <div class="flex flex-col sm:flex-row justify-between self-center">
         <h3 class="flex-grow font-medium"><%= quote.text %></h3>
         <span class="relative z-0 inline-flex rounded-md">
-          <%= link_to edit_quote_path(quote), class: 'inline-flex items-center p-1 border border-gray-300 text-xs font-medium rounded-l-md text-gray-900 bg-white hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-hamers-red-500' do %>
+          <%= unless quote.anonymous
```

Comment:
Mag iedereen alle quotes anonimiseren of alleen je eigen, aangezien dit niet ongedaan gemaakt kan worden?

Reply:
Momenteel iedereen

Reply:
Ik vind de change dat je het alleen zelf kan wel goed!

Reply:
Oh wacht ik heb gemaakt of admin of jezelf mag het, maar die knop staat er voor iedereen.